### PR TITLE
[#2140] Allow editor helper to accept minimum width and height for it's resizable elements

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2140-resizing-plugins-should-still-have-a-minimum-size

### CODE REVIEW

File: templates/assets/editors/editorhelper.js [hunk 1]
```diff
@@ -68,9 +70,15 @@
           if ( options.end ) {
             options.end();
           }
+
+          height = ( height / media.height ) * 100;
+          width = ( width / media.width ) * 100;
+          height = height >= options.minHeight ? height : options.minHeight;
+          width = width >= options.minWidth ? width : options.minWidth;
+
```
Comment (cadecairos) [inline templates/assets/editors/editorhelper.js:78]: The above 4 lines could be done in 2

File: templates/assets/editors/googlemap.js [hunk 2]
```diff
@@ -44,7 +45,13 @@ EditorHelper.addPlugin( "googlemap", function( trackEvent, popcornInstance ) {
       EditorHelper.draggable( trackEvent, container, media );
     }, false );
 
-    EditorHelper.resizable( trackEvent, container, media, "e, se, s, nw" );
+    options = {
```
Comment (cadecairos) [inline templates/assets/editors/googlemap.js]: this doesn't need to be a variable. just pass the object into the call to resizable.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @cadecairos received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/e759a2df3bc9a75/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @cadecairos received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/420c1dda12e7361/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/e759a2df3bc9a75/output.txt

Total script time: 1.89 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/420c1dda12e7361/output.txt

Total script time: 1.54 mins

Server is running on http://pr942.butyr.org/templates/basic/
Run the Test Suite: http://pr942.butyr.org/test/

File: templates/assets/editors/editorhelper.js (hunk 1 above)
Comment (cadecairos) [inline templates/assets/editors/editorhelper.js:78]: nevermind, I see whats you did here

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/5d31b918cfbd2fd/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/5d31b918cfbd2fd/output.txt

Total script time: 2.27 mins

Server is running on http://pr942.butyr.org/templates/basic/
Run the Test Suite: http://pr942.butyr.org/test/

File: templates/assets/editors/image/image.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ EditorHelper.addPlugin( "image", function( trackEvent ) {
       _container.innerHTML = "<span class=\"title\">Drag an image from your desktop</span>";
     }
 
-    window.EditorHelper.resizable( trackEvent, _container, _media );
+    window.EditorHelper.resizable( trackEvent, _container, _media, { minWidth: 25, minHeight: 25 } );
```
Comment (dseif) [inline templates/assets/editors/image/image.js]: multi line the object and spacing `} );` -> `});`

File: templates/assets/editors/wikipedia.js [hunk 4]
```diff
@@ -11,7 +11,7 @@ EditorHelper.addPlugin( "wikipedia", function( trackEvent ) {
 
   if ( window.jQuery ) {
 
-    window.EditorHelper.resizable( trackEvent, _container, media );
+    window.EditorHelper.resizable( trackEvent, _container, media, { minWidth: 40, minHeight: 40 } );
```
Comment (dseif) [inline templates/assets/editors/wikipedia.js]: same as above

**Pull request merged**